### PR TITLE
Introduce bibliographicLevel

### PR DESCRIPTION
**Pull request by TobiasNx**
Instead of a type "PartOfWork" I think we should introduce a new element `bibliographicLevel` derived from [Marc Leader Pos 07](https://www.loc.gov/marc/bibliographic/bdleader.html).

PartOfWork is not a reference to the content  as type provides.

Related to #1574

### CODE REVIEW

File: src/main/resources/alma/fix/mediumAndType.fix [hunk 1]
```diff
@@ -37,7 +53,10 @@ paste("@leaderTyp+008","@leaderTyp","008", join_char:"")
 # TODO: Check introx concerning possible mappings of 340 https://github.com/hbz/limetrans/blob/d2dff10e1b5cdf5699239a8f3474f8b652d582a3/src/main/resources/transformation/alma.fix#L629
 
 set_array("@300a_mainExtend")
-do list(path:"300  ", "var":"$i")
+do list(path:"300  ", "var":"$iContains fields that describe the subunit and data that identify the host item.
+
+Subunits may be items, folders, boxes, archival series, subgroups, or subcollections.
+")
```
Comment (blackwinter) [inline src/main/resources/alma/fix/mediumAndType.fix:59]: There seem to be multiple copy & paste (?) errors. Two more below.

Comment (TobiasNx) [reply src/main/resources/alma/fix/mediumAndType.fix]: Thanks

**Comment by acka47** [comment]
We should use controlled values not only strings or at east leave open the possibility to do so in the future. As justed discussed, we will use an object with `label` for now an add IDs for controlled values later.

**Review by acka47 — CHANGES_REQUESTED** [review]
See comment

**Comment by dr0i** [comment]
There are conflicts.

**Comment by TobiasNx** [comment]
I redid the last commit and merged the master into this branch before that.

**Comment by dr0i** [comment]
Will be deployed on Monday.

**Pull request merged**